app/assets/stylesheets/govuk_publishing_components/components/_search.scss

```diff
@@ -13,49 +13,99 @@ $large-input-size: 50px;
     @include govuk-font($size: 19, $line-height: $input-size);
     margin: 0;
   }
+
+  .js-enabled & {
+    @include govuk-font($size: 19, $line-height: $input-size);
```

Comment:
I know this wasn't part of your change but I think this line should be moved outside of the `.js-enabled` section here, as we want the font to be correct even if JS isn't enabled.